app/models/role.rb

```diff
@@ -0,0 +1,12 @@
+# == Schema Information
+#
+# Table name: roles
+#
+#  id          :bigint           not null, primary key
+#  description :string
+#  name        :string           not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+class Role < ApplicationRecord
+end
```

Comment:
Se debería agregar un validacion para `name` ya que es un campo not null:

` validates :name, presence: true`